opentech/apply/funds/templates/funds/includes/review_table.html

```diff
@@ -15,8 +15,12 @@
     {% if object.stage.has_external_review %}
         <tr class="tr tr--subchild"><td colspan="4"><hr></td></tr>
 
-        {% include 'funds/includes/review_table_row.html' with reviews=reviewer_reviews %}
-        {# Add logic for showing missing reviews #}
-        {% include 'funds/includes/review_table_row.html' with reviews=object.reviewers_not_reviewed missing=True %}
+        {% if reviewer_reviews or object.reviewers_not_reviewed  %}
+            {% include 'funds/includes/review_table_row.html' with reviews=reviewer_reviews %}
+            {% include 'funds/includes/review_table_row.html' with reviews=object.reviewers_not_reviewed missing=True class="hidden" %}
```

Comment:
shouldn't this and the toggle be wrapped in an if?
i.e. if there no more people who did not review, then there is no need for the toggle

Reply:
other than this, looks good to me